### DISCUSSION TITLE
Unable to Sync data between Cokreat and sunbirdED

### DISCUSSION THREAD

**Discussion by bharathravi-dev** [Sunbird-coKreat/Community#72] [General Discussions]
for the above functionality we are running 2 job Managers in our environment.
-> auto-creator-v2-jobmanager
->content-auto-creator-jobmanager
and there is no errors in logs of this jobmanager.

please help me on this functional flow.

@vinukumar-vs  @pallakartheekreddy

**Comment by mmanimegalai** [comment]
@BharathTarento Apologies, this got missed. Is this still an issue or sorted? cc @vaivk369 @Shubham Bansal @vinukumar-vs

**Reply by bharathravi-dev** [reply]
hi @mmanimegalai  we are still facing the  issues. please help me on this.

**Comment by mmanimegalai** [comment]
@vaivk369 @shubhambansaltarento Can you please help Bharath with the issue? cc @vinukumar-vs

**Comment by vaivk369** [comment]
@pallakartheekreddy can you help to confirm the jobs needed to run to sync the data?

**Reply by pallakartheekreddy** [reply]
@BharathTarento Where are below jobs running? Cokreat or sunbirdED?
1. auto-creator-v2-jobmanager
2. content-auto-creator-jobmanager
When you want to push content or question Set for coKreat to sunbirdED. Please make sure these jobs are running in sunbirdED.

**Reply by bharathravi-dev** [reply]
@pallakartheekreddy 
1.auto-creator-v2-jobmanager
2.content-auto-creator-jobmanager
it is running in both cokreat and sunbirdED, and we want to push content or question Set from coKreat to sunbirdED.
